google/cloud/pubsub/message.h

```diff
@@ -42,8 +43,9 @@ std::size_t MessageProtoSize(::google::pubsub::v1::PubsubMessage const& m);
 // For Open Telemetry tracing only. Inserts or sets an attribute on the message.
 void SetAttribute(std::string const& key, std::string value, pubsub::Message&);
 // For Open Telemetry tracing only. Gets the value for a given attribute key on
-// the message.
-std::string GetAttribute(std::string const& key, pubsub::Message& m);
+// the message. Note: the string_view is only valid for the lifetime of the
```

Comment:
You probably now want to mention what happens when `key` is not found.  For example, _"Returns the value for ..., **or the null string_view when not found**."_